apps/client/src/components/Form/FormElement.tsx

```diff
@@ -1,40 +1,24 @@
-import { Flex, FormLabel, Input } from "@chakra-ui/react";
-import styled from "@emotion/styled";
+import { Flex, FormLabel, Input, Text } from "@chakra-ui/react";
 import { useController } from "react-hook-form";
-import type { FormProps } from "~/components/Form/FormProps.type";
+import type { FormProps } from "./FormProps.type";
 
-export const FormElement = ({ control, name, label, type, rules, placeholder }: FormProps) => {
+export const FormElement = ({ control, name, label, type, ...rest }: FormProps) => {
   const {
     field,
     fieldState: { error: errors },
-  } = useController({ control, name, rules });
-
-  const isRequired = rules?.required;
-
-  const charCode = placeholder?.charCodeAt(placeholder.length - 1) as number;
-  const constantChar = (charCode - 44032) % 28;
+  } = useController({ control, name });
 
   return (
-    <Flex width="100%" position="relative" flexDir="column" rowGap="8px">
-      {label && (
-        <FormLabel htmlFor={name} fontSize="14px" margin="0">
-          {label} {isRequired && "*"}
-        </FormLabel>
+    <Flex width="100%" position="relative" flexDir="column">
```

Comment:
```suggestion
    <Flex width="full" position="relative" flexDir="column">
```

100%는 full로 통일하면 좋을 것 같습니다!